charts/kargo/templates/external-webhooks-server/deployment.yaml

```diff
@@ -82,42 +82,43 @@ spec:
           {{- toYaml . | nindent 8 }}
         {{- end }}
         ports:
-        - containerPort: 9443
-          name: webhooks-server
+        - containerPort: 8080
+          name: {{ if .Values.externalWebhooksServer.tls.enabled }}"https"{{ else }}"http"{{ end }}
           protocol: TCP
-        {{- if or .Values.externalWebhooksServer.tls.enabled .Values.kubeconfigSecrets.kargo  }}
+        {{- if or .Values.kubeconfigSecrets.kargo .Values.externalWebhooksServer.tls.enabled }}
         volumeMounts:
-        {{- if .Values.externalWebhooksServer.tls.enabled }}
-        - mountPath: /tmp/k8s-webhook-server/serving-certs
-          name: cert
+        - mountPath: /etc/kargo
+          name: config
           readOnly: true
         {{- end }}
-        {{- if .Values.kubeconfigSecrets.kargo }}
-        - mountPath: /etc/kargo/kubeconfigs
-          name: kubeconfigs
-          readOnly: true
-        {{- end }}
-        {{- end }}
         {{- with .Values.externalWebhooksServer.securityContext | default .Values.global.securityContext }}
         securityContext:
           {{- toYaml . | nindent 10 }}
         {{- end }}
         resources:
           {{- toYaml .Values.externalWebhooksServer.resources | nindent 10 }}
-      {{- if or .Values.externalWebhooksServer.tls.enabled .Values.kubeconfigSecrets.kargo  }}
+      {{- if or .Values.kubeconfigSecrets.kargo .Values.externalWebhooksServer.tls.enabled }}
       volumes:
```

Comment:
All volumes/volumeMounts have been updates to more closely resemble the API server instead of the admission control webhooks server.

Reply:
Without these changes, the external webhooks server was failing to find its cert when TLS is enabled.